form/iframe-form.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useCallback, useRef } from 'react'
+
+import { Card } from '../design'
+
+const IframeForm = ({ className, title, ...passProps }) => {
+  const ref = useRef(null)
+  const resize = useCallback(() => {
+    ref.current.style.height = `${ref.current.contentWindow.document.body.offsetHeight}px`
+  }, [])
+
+  useEffect(() => {
+    const observer = new ResizeObserver(resize)
+    observer.observe(ref.current.contentWindow.document.body)
+    return () => observer.disconnect()
+  }, [])
```

Comment:
Should not this depend on `ref` or `ref.current`?

Reply:
ref is created only once.